gensrc/thrift/InternalService.thrift

```diff
@@ -266,6 +267,11 @@ struct TQueryOptions {
   115: optional TTimeUnit big_query_profile_threshold_unit = TTimeUnit.SECOND;
   
   116: optional string sql_dialect;
+
+  117: optional bool enable_spill_to_remote_storage;
+  118: optional list<string> spill_remote_storage_paths;
+  119: optional CloudConfiguration.TCloudConfiguration spill_remote_storage_conf;
+  120: optional bool disable_spill_to_local_disk;
```

Comment:
can merge 118 and 120 to 119?

Reply:
> can merge 118 and 120 to 119?

good suggestion, I merge them into TSpillToRemoteStorageOptions